src/ConnectedKubernetes/ConnectedKubernetes.Autorest/docs/Set-AzConnectedKubernetes.md

```diff
@@ -98,7 +154,61 @@ Location Name              ResourceGroupName
 eastus   azps_test_cluster azps_test_group
 ```
 
-This command disable gateway feature of a connected kubernetes.
+This command enables gateway feature of a connected kubernetes.
+
+### Example 3: Enable gateway feature of connected kubernetes with InputObject
+```powershell
+# Get an existing clustser first
+$inputObject = Get-AzConnectedKubernetes -ClusterName azps_test_cluster -ResourceGroupName azps_test_group -SubscriptionId $subscriptionId
+# Enable gateway and set gateway resource Id 
+$outputObject.GatewayEnabled=$true
+$outputObject.GatewayResourceId=$gatewayResourceId
+Set-AzConnectedKubernetes -InputObject $inputObject     
```

Comment:
Wouldn't you do this by making a call like this?

```PowerShell
Get-AzConnectedKubernetes -ClusterName azps_test_cluster -ResourceGroupName azps_test_group -SubscriptionId $subscriptionId
| Set-AzConnectedKubernetes -GatewayResourceId $gatewayResourceId
```

Reply:
(similar applies to other examples below - they all look fine, but we introduced `-WorkloadIdentityDisabled` to make this possible)